unime/src/routes/+layout.svelte

```diff
@@ -39,23 +39,14 @@
 
   const systemColorScheme = window.matchMedia('(prefers-color-scheme: dark)');
 
-  systemColorScheme.addEventListener(
```

Comment:
By removing this, you remove the listener to window events completely. When the theme is now changed through the OS, it does not have any effect anymore.